src/index.ts

```diff
@@ -238,6 +239,7 @@ export function createAlchemyWeb3(
   patchEnableCustomRPC(alchemyWeb3);
   patchEthFeeHistoryMethod(alchemyWeb3);
   patchEthMaxPriorityFeePerGasMethod(alchemyWeb3);
+  patchEthGasStation(alchemyWeb3);
```

Comment:
As written, the new method isn't exposed in the TypeScript types. We'll want to update the definition of `AlchemyWeb3` to include the new method.